src/view.rs

```diff
@@ -331,6 +347,7 @@ pub fn view_menu_bar(m: &Model) -> Html {
             //<>
                 { nest_grid_button }
             //</>
+                { nest_row_input } { nest_col_input }
```

Comment:
this is also a good idea. but i wonder if it'll render differently. can you post a gif of the zoomed in button?